Community-Supported/publish-hyper/publish-hyper-file.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+import tableauserverclient as TSC
+from tableauhyperapi import HyperProcess, Telemetry, \
+    Connection, CreateMode, \
+    NOT_NULLABLE, NULLABLE, SqlType, TableDefinition, \
+    Inserter, \
+    escape_name, escape_string_literal, \
+    TableName, \
+    HyperException
+
+
+# Configure for TSC to publish
+# Note: Do not store creds/tokens in plaintext, please use env vars :)
+hyper_name = 'customer.hyper'
+server_address = 'https://10ax.online.tableau.com/'
+site_name = 'mysitename'
+project_name = 'projectname'
+token_name = 'tokennamehere'
+token_value = 'tokenherebutpleaseconsidersecuritypolicies'
+
+path_to_database = Path(hyper_name)
+
+# The table is called "Extract" and will be created in the "Extract" schema
+# and contains four columns.
+extract_table = TableDefinition(
+    table_name=TableName("Extract", "Extract"),
+    columns=[
+        TableDefinition.Column(name='Customer ID', type=SqlType.text(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Customer Name', type=SqlType.text(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Loyalty Reward Points', type=SqlType.big_int(), nullability=NOT_NULLABLE),
+        TableDefinition.Column(name='Segment', type=SqlType.text(), nullability=NOT_NULLABLE)
+    ]
+)
+
+
+def insert_data():
```

Comment:
Can you add a comment + link that this function is taken from the single table creation example?